modules/sap_hana/outputs.tf

```
@@ -15,13 +15,13 @@
  */
 
 output "instance_name" {
-  value = "${google_compute_instance.gcp_sap_hana.name}"
+  value = "${google_compute_instance.gcp_service_project_vm.name}"
```

Outputs must have descriptions.

```suggestion
  value       = google_compute_instance.gcp_service_project_vm.name
  description = "The name of the SAP VM instance."
```

---

This still needs to be addressed.